gradio/blocks.py

```diff
@@ -768,28 +769,32 @@ def clear(self):
         self.children = []
         return self
 
-    def configure_queue(
+    @document()
+    def queue(
         self,
-        live_queue_updates: bool = True,
         concurrency_count: int = 1,
-        data_gathering_start: int = 30,
-        update_intervals: int = 5,
-        duration_history_size: int = 100,
+        status_update_rate: float | str = "auto",
+        client_position_to_load_data: int = 30,
+        default_enabled: bool = True,
     ):
         """
+        You can control the rate of processed requests by creating a queue. This will allow you to set the number of requests to be processed at one time, and will let users know their position in the queue.
         Parameters:
-            live_queue_updates: If true, Queue will send estimations to clients whenever a job is finished. Otherwise will send estimations periodically, might be preferred when events have very short process-times.
-            concurrency_count: Number of max number concurrent jobs inside the Queue.
-            data_gathering_start: If Rank<Parameter, Queue asks for data from the client. You may make it smaller if users can send very big sized data (video or such) to not overflow the memory.
-            update_intervals: Queue will send estimations to the clients in intervals=update_intervals when live_queue_updates==false
-            duration_history_size: Queue duration estimation calculation window size.
+            concurrency_count: Number of worker threads that will be processing requests concurrently.
+            status_update_rate: If "auto", Queue will send status estimations to all clients whenever a job is finished. Otherwise Queue will send status at regular intervals set by this parameter as the number of seconds.
+            client_position_to_load_data: Once a client's position in Queue is less that this value, the Queue will collect the input data from the client. You may make this smaller if clients can send large volumes of data, such as video, since the queued data is stored in memory.
+            default_enabled: If True, all event listeners will use queueing by default.
+        Example:
+            demo = gr.Interface(gr.Textbox(), gr.Image(), image_generator)
```

Comment:
use string shortcuts in Interface for simplicity

Reply:
I think we should not use string shortcuts cuz we removed them from our docs and we just support backwards compatibility.

Reply:
Agree with @FarukOzderim.